aio/content/guide/docs-style-guide.md

```diff
@@ -1144,25 +1141,25 @@ Here's the `src` attribute for the "flying hero" image belonging to this page.
 src="generated/images/guide/docs-style-guide/flying-hero.png"
 ```
 
-<h3 class="no-toc">Use the HTML <i>&lt;img&gt;</i> tag</h3>
+<h3 class="no-toc">Use the HTML `&lt;img&gt;` tag</h3>
```

Comment:
gah! Also an accident - not worth explaining how I keep doing this, but thank you for catching it. I meant: `<img>` since we can render code in headers now. Fixed.